restapi/src/main/java/io/stargate/web/docsapi/resources/DocumentResourceV2.java

```diff
@@ -762,18 +759,20 @@ public Response searchDoc(
 
           DocumentDB db = dbFactory.getDocDataStoreForToken(authToken, getAllHeaders(request));
 
+          int docPageSize = Math.max(1, pageSizeParam);
```

Comment:
Looks like Paginator already does this? 
https://github.com/stargate/stargate/blob/master/restapi/src/main/java/io/stargate/web/docsapi/dao/Paginator.java#L32